setup.py

```diff
@@ -9,8 +9,8 @@
     packages=find_packages(),
     install_requires=[
         'cdislogging',
-        'Flask==0.10.1',
-        'Werkzeug==0.12.2',
+        'Flask',
```

Comment:
I think you can just remove these lines from setup.py altogether. If a client application installs this package and doesn't specify a pin, these will pull the latest version on pypi.